terraform/domain.tf

```diff
@@ -79,4 +79,12 @@ resource "aws_route53_record" "docs" {
   type    = "CNAME"
   ttl     = "300"
   records = ["lincbrain.github.io."]
+}
+
+resource "aws_route53_record" "hub" {
+  zone_id = aws_route53_zone.linc-brain-mit.zone_id
+  name    = "hub"
+  type    = "CNAME"
+  ttl     = "300"
+  records = ["hub.dandiarchive.org."]
 }
```

Comment:
I think we can actually remove this since `hub.dandiarchive.org` already exists in `dandiarchive.org` hosted zone in AWS

Reply:
Perhaps I implemented this incorrectly, but I was trying to add a redirect from `hub.lincbrain.org` to `hub.dandiarchive.org` since `hub.lincbrain.org` is no longer live.

Reply:
A CNAME is usually not the method of doing a redirect -- what you'd need is someone to visit `hub.lincbrain.org` to receive a 302 HTTP Redirect to `hub.dandiarchive.org` (thus you'd need some form of nginx proxy running etc)

I think it is fine to just remove the DNS record for `hub.lincbrain.org`

Reply:
Thanks for clarifying.  That sounds good.